src/mds/MDBalancer.cc

```diff
@@ -169,6 +171,28 @@ mds_load_t MDBalancer::get_load(utime_t now)
   return load;
 }
 
+int MDBalancer::localize_balancer(string const balancer)
+{
+  int64_t pool_id = mds->mdsmap->get_metadata_pool();
+  string fname = "/tmp/" + balancer;
```

Comment:
Does it have to go into a /tmp/ file?  I would expect these scripts are usually small enough to painlessly hold in memory after we've read it from rados